File: src/main/java/com/github/ruslanmn/springbootchat/fake/FakeRest.java
package com.github.ruslanmn.springbootchat.fake;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.security.core.Authentication;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class FakeRest {

    @GetMapping("/fakerest")
    public String helloWorld() {
        return "Hello world";
    }

    @GetMapping("/hello")
    public String chat(Authentication authentication) {
        return "Hello, " + authentication.getName();
    }
}

File: src/main/kotlin/com/github/ruslanmn/springbootchat/fake/FakeRest.kt
package com.github.ruslanmn.springbootchat.fake

import org.springframework.web.bind.annotation.RestController
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker
import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer
import org.springframework.messaging.simp.config.MessageBrokerRegistry
import org.springframework.web.socket.config.annotation.StompEndpointRegistry
import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
import org.springframework.beans.factory.annotation.Autowired
import kotlin.Throws
import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder
import org.springframework.security.config.annotation.web.builders.HttpSecurity
import org.springframework.security.crypto.password.PasswordEncoder
import lombok.AllArgsConstructor
import org.springframework.messaging.handler.annotation.MessageMapping
import org.springframework.messaging.handler.annotation.SendTo
import org.springframework.web.util.HtmlUtils
import org.springframework.boot.autoconfigure.SpringBootApplication
import kotlin.jvm.JvmStatic
import org.springframework.boot.SpringApplication
import org.springframework.security.core.Authentication

@RestController
class FakeRest {
    @GetMapping("/fakerest")
    fun helloWorld(): String {
        return "Hello world"
    }

    @GetMapping("/hello")
    fun chat(authentication: Authentication): String {
        return "Hello, " + authentication.name
    }
}